[Bug]: Invalid import in the Typescript ESM installation guide.

### Version

14.4.2

### Steps to reproduce

1. Go to the page https://thymikee.github.io/jest-preset-angular/docs/getting-started/installation#customizing
2. Select `Typescript ESM` tab 

### Expected behavior

This code should not generate any errors.

### Actual behavior

![image](https://github.com/user-attachments/assets/5dc99338-3d8e-4acb-8b11-7e366465338e)

TS2307: Cannot find module 'jest-preset-angular/setup-env/zone.mjs' or its corresponding type declarations.

### Additional context

_No response_

### Environment

```shell
N/A
```

--- a/website/docs/getting-started/installation.md
+++ b/website/docs/getting-started/installation.md
@@ -121,7 +121,7 @@ setupZoneTestEnv();
 ```
 
 ```ts title="setup-jest.ts" tab={"label":"TypeScript ESM"}
-import { setupZoneTestEnv } from 'jest-preset-angular/setup-env/zone.mjs';
+import { setupZoneTestEnv } from 'jest-preset-angular/setup-env/zone/index.mjs';
 import './jest-global-mocks';
 
 setupZoneTestEnv();

--- a/website/versioned_docs/version-14.3/getting-started/installation.md
+++ b/website/versioned_docs/version-14.3/getting-started/installation.md
@@ -121,7 +121,7 @@ setupZoneTestEnv();
 ```
 
 ```ts title="setup-jest.ts" tab={"label":"TypeScript ESM"}
-import { setupZoneTestEnv } from 'jest-preset-angular/setup-env/zone.mjs';
+import { setupZoneTestEnv } from 'jest-preset-angular/setup-env/zone/index.mjs';
 import './jest-global-mocks';
 
 setupZoneTestEnv();

--- a/website/versioned_docs/version-14.4/getting-started/installation.md
+++ b/website/versioned_docs/version-14.4/getting-started/installation.md
@@ -121,7 +121,7 @@ setupZoneTestEnv();
 ```
 
 ```ts title="setup-jest.ts" tab={"label":"TypeScript ESM"}
-import { setupZoneTestEnv } from 'jest-preset-angular/setup-env/zone.mjs';
+import { setupZoneTestEnv } from 'jest-preset-angular/setup-env/zone/index.mjs';
 import './jest-global-mocks';
 
 setupZoneTestEnv();

